frontend/public/components/service.jsx

```diff
@@ -142,121 +103,139 @@ const ServiceAddress = ({ s }) => {
     switch (type) {
       case 'NodePort':
         return ServiceIPsRow(
-          'Node Port',
-          'Accessible outside the cluster',
+          t('network-service~Node port'),
+          t('network-service~Accessible outside the cluster'),
           _.map(s.spec.ports, 'nodePort'),
-          '(all nodes): ',
+          t('network-service~(all nodes): '),
         );
       case 'LoadBalancer':
         return ServiceIPsRow(
-          'External Load Balancer',
-          'Ingress point(s) of load balancer',
+          t('network-service~External load balancer'),
+          t('network-service~Ingress point(s) of load balancer'),
```

Comment:
nit: Not new, but I would remove the parentheses around the `s`

Reply:
fixed